ui-v2/app/templates/application.hbs

```diff
@@ -1,11 +1,11 @@
 {{#if (not loading)}}
-    {{outlet}}
+  {{outlet}}
 {{else}}
-{{#hashicorp-consul id="wrapper" dc=dc nspaces=nspaces nspace=nspace}}
+  {{#hashicorp-consul id="wrapper" permissions=permissions dcs=dcs dc=dc nspaces=nspaces nspace=nspace}}
```

Comment:
Are the permissions expected to be empty here? If not, I only saw them added to the dc route model - how do they get fetched / populated here?

Reply:
Ahhh, good spot - these aren't getting set. It doesn't matter too much as this is only for the loading page, which means if you are trying to look for the 'Manage namespaces' button in the namespace dropdown whilst a page is loading, then you won't find it. But also we can't tell if you can view this button as we need to load the API endpoint to check - so we show the loading page! 🐔 🥚 

Thing is though, remember I was asking about all the jumping around in the menu when pages where loading - I have a feeling this might be the root of it! I'm going to merge this s is as I don't think this is a major problem and it would be good to get more user testing done here. but I'm definitely gonna take a deeper look at this. We'll probably look at doing what it says in this TODO:

https://github.com/hashicorp/consul/blob/fce23da02531da9944bd25ae1711473ed5091a9e/ui-v2/app/routes/dc.js#L5-L6